Dockerfile

```diff
@@ -1,17 +1,13 @@
-# The bitnami images are stripped down and offer things like install_packages
-# that avoid crufting up the image with lots of dependencies. 
-# 
-FROM bitnami/node:8.11.3
-#FROM circleci/node:8-stretch-browsers
+FROM node:8.11.3
 
-RUN install_packages libsecret-1-dev libglib2.0-dev
+# The key bits here are making sure we install, libsecret-1-dev libglib2.0-dev
+RUN apt-get update -qq && apt-get install -y \
+  libsecret-1-dev libglib2.0-dev && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
What’s this for, to reduce the size of the resulting image?

Reply:
Yes, it's typical to do it all in one `RUN` to reduce the size of that step's layer. It's what bitnami's `install_package` does.

Reply:
Thanks 👌